File: app/src/main/java/com/nollpointer/dates/WikipediaApi.java
package com.nollpointer.dates;

import retrofit2.Call;
        import retrofit2.http.GET;
        import retrofit2.http.Path;

public interface WikipediaApi {
    @GET("/api/rest_v1/page/summary/{title}")
    Call<WikipediaResponseModel> getData(@Path("title") String title);
}

File: app/src/main/java/com/nollpointer/dates/api/WikipediaApi.kt
package com.nollpointer.dates.api

import com.nollpointer.dates.model.WikipediaResponseModel
import io.reactivex.Single
import retrofit2.http.GET
import retrofit2.http.Path

/**
 * Api для работы с Википедией
 *
 * @author Onanov Aleksey (@onanov)
 */
interface WikipediaApi {
    @GET("page/summary/{title}")
    fun getData(@Path("title") title: String?): Single<WikipediaResponseModel>
}